README.md

```diff
@@ -1,23 +1,27 @@
 Formality
 =========
 
-Formality is a proof and programming language, as well as a standard format for
-algorithms and proofs. It exists to fill a hole in the current market: there
-aren't many programming languages featuring theorem proving that are 1.
-portable, 2. efficient, 3. accessible.
-
-**Portability** makes it available everywhere, like a JSON of code. We
-accomplish that by bootstrapping: we design a minimal core that can be quickly
-implemented anywhere, and build the rest of the language on top of it.
-
-**Efficiency** is required if we want the language to be used for production
-software. We accomplish that by compiling Formality to a massively parallel,
+Formality is a dependently-typed programming language that enables developers to
+use formal proofs to make their applications safer, faster and friendlier.
+
+## Design Principles
+
+**Portable:** Available everywhere, like a type-safe JSON for code. Formality
+has a minimal core that bootstraps the language and can be quickly implemented
+anywhere.
+
+**Efficient:** On the web, Formality's JavaScript compiler is [competitive with
+native JS
+code](https://medium.com/@maiavictor/the-refreshing-simplicity-of-compiling-formality-to-anything-388a1616f36a).
+On the blockchain, the EVM compiler is also extremely performant at
+approximately [270 gas per
+beta-reduction](https://medium.com/@maiavictor/compiling-formality-to-the-evm-99aec75677dd).
+Additionally, Formality can compile to a new massively parallel,
```

Comment:
First problem: I'd like to avoid any content that would require update in a future. Also, this paper should have no external dependencies on sites like medium. If the 270 gas per beta-reduction number changes, we'll need to update the paper.

Second problem: I'd like to stress that this paper should be seen as a foundational, purely mathematical one. It should have no external dependencies or citations. It is the root. External links are not allowed. Mentioning companies or people isn't allowed. Not even specific technology is allowed (Bitcoin, Ethereum). So, a HTTP link to a privately owned company is absolutely no ok.

Note there are places where I do mention names (ex: murmurhash). That should be fixed when possible.

Reply:
If we mention performance we should have *some* benchmarks somewhere in the paper, I will remove specific figures from this section and make links to placeholders in the Formality-Comp section